administrator/components/com_joomlaupdate/src/Model/UpdateModel.php

```diff
@@ -212,7 +235,7 @@ public function getCheckForSelfUpdate()
 
             try {
                 $joomlaUpdateComponentObject = $db->loadObject();
-            } catch (\RuntimeException $e) {
+            } catch (RuntimeException $e) {
```

Comment:
```suggestion
            } catch (\RuntimeException $e) {
```
Revert this change because in J4 the file is namespaced.